src/entities/explorer/ui/explorer-docs/docs-details/index.tsx

```diff
@@ -0,0 +1,51 @@
+import ArrowRightIcon from '@mui/icons-material/ArrowRight';
+import Grid from '@mui/material/Grid';
+import { useMemo } from 'react';
+
+import { useExplorer, useExplorerProps, Options } from '../../../hooks';
+import { IDocsTypeDetails as IExplorerDocsDetailsProps } from '../../../model';
+import styles from '../styles.module.scss';
+
+export const ExplorerDocsDetails = ({ typeDetails }: IExplorerDocsDetailsProps): JSX.Element => {
+  const { selectedElements, setSelectedElements, setFieldInfo } = useExplorer();
+  const docsDetails = useExplorerProps({ parsedField: typeDetails, option: Options.details });
+
+  const findFieldInfo = useMemo(() => {
+    return (name: string) => {
+      return typeDetails.args.find((value) => value.name === name);
+    };
+  }, [typeDetails]);
```

Comment:
I don't get it why you memoize function with useMemo? For functions there is `useCallback`